src/waveform/renderers/waveformmark.cpp

```diff
@@ -8,7 +8,8 @@
 #include "widget/wskincolor.h"
 
 WaveformMark::WaveformMark()
-    : m_pPointCos(nullptr) {
+    : m_pPointCos(nullptr),
```

Comment:
This deletes the implicit WaveformMark copy operator, which is used by QList in WaveformMarkSet.
I guess making the list of WaveformMarkSet store pointers of WaveformMarks instead of WaveformMarks won't work either, because unique_ptr can't be copied. Am I wrong?